components/infobox/wikis/apexlegends/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,200 @@
+	['analyst'] = {category = 'Analysts', variable = 'Analyst', isplayer = false},
+	['observer'] = {category = 'Observers', variable = 'Observer', isplayer = false},
+	['host'] = {category = 'Hosts', variable = 'Host', isplayer = false},
+	['journalist'] = {category = 'Journalists', variable = 'Journalist', isplayer = false},
+	['expert'] = {category = 'Experts', variable = 'Expert', isplayer = false},
+	['coach'] = {category = 'Coaches', variable = 'Coach', isplayer = false},
+	['caster'] = {category = 'Casters', variable = 'Caster', isplayer = false},
+	['talent'] = {category = 'Talents', variable = 'Talent', isplayer = false},
+	['manager'] = {category = 'Managers', variable = 'Manager', isplayer = false},
+	['producer'] = {category = 'Producers', variable = 'Producer', isplayer = false},
+	['admin'] = {category = 'Admins', variable = 'Admin', isplayer = false},
+}
+local _SIZE_LEGEND = '25x25px'
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+
+	if String.isEmpty(player.args.team) then
+		player.args.team = PlayerTeamAuto._main{team = 'team'}
+	end
+
+	if String.isEmpty(player.args.team2) then
+		player.args.team2 = PlayerTeamAuto._main{team = 'team2'}
+	end
+
+	player.args.history = tostring(TeamHistoryAuto._results{addlpdbdata='true'})
+
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.createBottomContent = CustomPlayer.createBottomContent
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+	player.defineCustomPageVariables = CustomPlayer.defineCustomPageVariables
+
+	_args = player.args
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'region' then
+		return {}
+	elseif id == 'status' then
+		return {
+			Cell{name = 'Status', content = CustomPlayer._getStatusContents()},
+			Cell{name = 'Years Active (Player)', content = {_args.years_active}},
+			Cell{name = 'Years Active (Org)', content = {_args.years_active_manage}},
+			Cell{name = 'Years Active (Coach)', content = {_args.years_active_coach}},
+		}
+	elseif id == 'role' then
+		return {
+			Cell{name = 'Role', content = {
+				CustomPlayer._createRole('role', _args.role),
+				CustomPlayer._createRole('role2', _args.role2)
+			}},
+		}
+	elseif id == 'history' then
+		table.insert(widgets, Cell{
+			name = 'Retired',
+			content = {_args.retired}
+		})
+	end
+	return widgets
+end
+
+function CustomInjector:addCustomCells(widgets)
+	-- Signature Legends
+	table.insert(widgets,
+		Builder{
+			builder = function()
+				local legendIcons = Array.map(Player:getAllArgsForBase(_args, 'legends'),
+					function(legend, _)
+						return LegendIcon.getImage{legend, size = _SIZE_LEGEND}
+					end
+				)
+				return {
+					Cell{
+						name = #legendIcons > 1 and 'Signature Legends' or 'Signature Legend',
+						content = {
+							table.concat(legendIcons, '&nbsp;')
+						}
+					}
+				}
+			end
+		})
```

Comment:
not sure using builder here is really good
could just do it without and do the table insert on the cell instead

also if no legendsX params are set this might render an empty cell (on the right)